homeassistant/components/synology_dsm/media_source.py

```diff
@@ -223,13 +233,16 @@ async def get(
         # location: {cache_key}/{filename}
         cache_key, file_name = location.split("/")
         image_id = int(cache_key.split("_")[0])
+        shared = False
+        if file_name.endswith("_shared"):
+            shared = True
```

Comment:
```suggestion
        if shared := file_name.endswith("_shared"):
```

we can use the [walrus operator](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions) in this case